.github/workflows/ci.yml

```diff
@@ -47,7 +47,6 @@ jobs:
           # FIXME(issue #2138): run wasm tests, failing to run since https://github.com/mthom/scryer-prolog/pull/2137 removed wasm-pack
           - { os: ubuntu-22.04,   rust-version: nightly, target: 'wasm32-unknown-unknown',   publish: true, args: '--no-default-features' , test-args: '--no-run --no-default-features' }
           # rust versions
-          - { os: ubuntu-22.04,   rust-version: "1.70",  target: 'x86_64-unknown-linux-gnu'}
```

Comment:
I think this job should have been changed to use 1.77 instead of being removed. This job is here to test that the version declared in the Cargo.toml still works, which I think is what to `# rust version` comment directly above it tried to convey.